lib/features/composer/presentation/composer_controller.dart

```diff
@@ -191,6 +192,7 @@ class ComposerController extends BaseController with DragDropFileMixin implement
   ButtonState _closeComposerButtonState = ButtonState.enabled;
   ButtonState _saveToDraftButtonState = ButtonState.enabled;
   ButtonState _sendButtonState = ButtonState.enabled;
+  bool _keepEditedIdentityContentOnOpen = true;
```

Comment:
I still not understand the idea of this

Reply:
On opening draft or cache composer, signature content may have been edited. At the moment of applying identity, the edited signature will be overridden. This give the signature one chance to prevent that from happening.

Reply:
please use enum for this

Reply:
Done